turbinia/workers/partitions.py

```diff
@@ -123,6 +129,10 @@ def _ProcessPartition(self, path_spec):
 
       path_spec = path_spec.parent
 
+    log.debug(
+        'Partition processing found path_specs {0!s} for partition '
+        'location {1!s} index {2!s}'.format(
+            path_spec_chain, location, partition_index))
```

Comment:
This is fine, but may not need the `partition_index`. Not all partitions will have one.

Reply:
Good catch.  I think I'll leave this in for now though just for the extra information.